src/org/usfirst/frc/falcons6443/robot/commands/autocommands/DriveToDistanceStop.java

```diff
@@ -6,7 +6,8 @@
 
 public class DriveToDistanceStop extends CommandGroup {
     public DriveToDistanceStop(int distance){
-            addSequential(new DriveToDistance(distance));
-            addSequential(new StopDrive());
+            addParallel(new DriveToDistance(distance)); //all three commands run at the same time
+            addParallel(new MoveTurret());
```

Comment:
I answered the question above

Reply:
When does the addParallel Command stop running? When the sequential under it is finished, or when the Command linked to addParallel stops? Like, would DriveToDistance stop when MoveShooter is done, or does it stop when it gets to the right distance?

Reply:
It is done running when the addSequential beneath it is finished or if the addParallel command is longer, it will finish running when the addParallel command is finished. Essentially it will finish whenever the longest command is finished.